Auth0.podspec

```diff
@@ -41,6 +41,7 @@ Pod::Spec.new do |s|
   s.source           = { :git => 'https://github.com/auth0/Auth0.swift.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/auth0'
   s.source_files     = 'Auth0/*.swift'
+  s.resources        = "Sources/AuthFoundation/Resources/**/*"
```

Comment:
The path `Sources/AuthFoundation/Resources/**/*` does not exist, and as such Cocoapods cannot find the file. Did you test the changes in a sample app that uses Cocoapods?

<img width="645" alt="Screenshot 2024-04-12 at 13 55 01" src="https://github.com/auth0/Auth0.swift/assets/5055789/aec53dd4-794e-4d54-b95a-28287b320a9b">